runtime/altair/src/xcm.rs

```diff
@@ -77,7 +88,7 @@ parameter_types! {
 	pub AirPerSecond: (AssetId, u128) = (
 		MultiLocation::new(
 			1,
-			X2(Parachain(parachains::altair::ID), GeneralKey(parachains::altair::AIR_KEY.to_vec())),
+			X2(Parachain(ParachainInfo::parachain_id().into()), GeneralKey(parachains::altair::AIR_KEY.to_vec())),
```

Comment:
Now that we can test other tokens using asset-registry, we no longer need to mock altair as another chain to transfer air around, so we no longer need to hardcode altair's parachain Id.